settings.py

```diff
@@ -22,3 +22,18 @@
 to be at least greater than FLOW_STATS and ideally it slightly greater than
 whichever longest network convergence FlowMods operations that your network has.
 """
+
+ACTION_PACES = {
+    "send_flow_mod": {
+        "pace": "100/second",
+        "strategy": "fixed_window",
+    },
+    "send_flow_mod.no_owner": {
+        "pace": "50/second",
+        "strategy": "fixed_window",
+    },
+    "send_flow_mod.mef_eline": {
+        "pace": "50/second",
```

Comment:
`mef_eline` current equivalent pacing interval is supposed to be `"100/second"`

It has `BATCH_INTERVAL = 0.5` and `BATCH_SIZE = 50`

https://github.com/kytos-ng/mef_eline/blob/master/settings.py#L31-L38

Can you review this again?

Reply:
Updated to 100/second.